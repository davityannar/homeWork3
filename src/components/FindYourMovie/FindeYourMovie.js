import React from 'react';

import './FindeYourMovie.scss';

const FindeYourMovie = () => {
    return(
        <p className='find-your-movie'>
            Find your movie
        </p>
    )
}

export default FindeYourMovie;