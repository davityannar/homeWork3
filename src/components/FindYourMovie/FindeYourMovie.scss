.find-your-movie {
    margin: 20px 0;
    text-transform: uppercase;
    color: #fff;
    font-weight: bold;
}