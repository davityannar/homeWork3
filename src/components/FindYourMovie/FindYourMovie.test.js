import React from 'react'
import FindYourMovie from './FindeYourMovie'

describe('Tests FindYourMovie component', () => {
  it('Matching snapshot', () => {
    const component = shallow(<FindYourMovie />);

    expect(component).toMatchSnapshot();
  })
})