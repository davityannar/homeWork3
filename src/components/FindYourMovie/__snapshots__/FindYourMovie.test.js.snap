// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests FindYourMovie component Matching snapshot 1`] = `
<p
  className="find-your-movie"
>
  Find your movie
</p>
`;
